server/app/services/question/types/QuestionDefinitionBuilder.java

```diff
@@ -67,7 +67,6 @@ public static QuestionDefinitionBuilder sample(QuestionType questionType) {
             .setName("")
             .setDescription("")
             .setQuestionText(LocalizedStrings.of(Locale.US, "Sample question text"))
-            .setQuestionHelpText(LocalizedStrings.of(Locale.US, "Sample question help text"))
```

Comment:
Can we add a unit test to check if the help text is blank?

Reply:
Done!